packages/webview_flutter/webview_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.9.1
+
+* Fixes unnecessary imports after [adding some exports of public API in foundation/serialization.dart](https://github.com/flutter/flutter/pull/104231).
```

Comment:
It doesn't fix any unnecessary imports, so this description is misleading. How about `Ignores unnecessary import warnings in preparation for [upcoming Flutter changes](https://github.com/flutter/flutter/pull/104231).`